scripts/install-with-latest-dependencies

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Is this script getting called somewhere? Maybe accidentally committed?

Reply:
To get more concrete, I still had `pluggy=1.3.0` locally and wouldn't have been able to reproduce this problem until I upgraded it (it's a transitive dependency of `pytest`)